src/main/java/org/folio/innreach/domain/entity/CentralServer.java

```diff
@@ -153,6 +155,10 @@
   )
   private List<LocalAgency> localAgencies = new ArrayList<>();
 
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "inn_reach_recall_user_id")
+  private InnReachRecallUser innReachRecallUser;
```

Comment:
If I understand the requirements correctly, it should be one to one relation. Also, is there any use case to make it part of CentralServer model? If yes, then I believe the join fetch queries need to be updated to load this entity as well

Reply:
There's nothing about relation in the requirements, but I asked Brooks.
Regarding the models - it depends on relation

Reply:
There are more details in related UI story, it's not clearly specified as requirement, but from the mock-ups it looks like there will be only recall user per central server)